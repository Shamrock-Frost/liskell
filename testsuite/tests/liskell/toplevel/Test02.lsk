;; Dependency tracking test for GHC

(defmodule Test02 _ (Prelude Test02Dep))

(define main undefined)

