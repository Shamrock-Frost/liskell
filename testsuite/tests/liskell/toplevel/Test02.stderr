[1 of 2] Compiling Test02Dep        ( Test02Dep.lsk, Test02Dep.o )
[2 of 2] Compiling Test02           ( Test02.lsk, Test02.o )
Linking Test02 ...
